spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -3,8 +3,21 @@
     given_there_is_an_edition
     when_i_go_to_edit_the_edition
     and_i_go_to_insert_an_attachment
+    and_i_select_a_file
     and_i_upload_a_file_attachment
     then_i_can_see_previews_of_the_attachment
+    and_i_can_see_a_link_to_the_attachment
```

Comment:
We should reconsider these steps, as they're actually referring to the two different kinds of insertion methods for an inline file attachment, which doesn't make sense for featured file attachments at this point in the workflow. Can we have two distinct steps - something like `then_i_can_insert_the_attachment_inline` and `then_i_can_see_the_attachment`.

Reply:
This step no longer exists.